batchupload/uploader.py

```diff
@@ -2,13 +2,14 @@
 # -*- coding: utf-8  -*-
 #
 from __future__ import unicode_literals
-from builtins import open
 import batchupload.common as common
 import batchupload.prepUpload as prepUpload
+from batchupload.make_info import make_info_page
 import os
 import pywikibot
 
 FILE_EXTS = ('.tif', '.jpg', '.tiff', '.jpeg')
```

Comment:
This is actually just a default set of formats for "photo" type images. All the methods which make use of this are designed to allow you to override the selection. So e.g. for KMB I would probably only pass `('.jpg', '.jpeg')` since TIFF images are unexpected and should trigger a warning.